lib/cf_obs_binary_builder/dependencies/libgdiplus.rb

```diff
@@ -0,0 +1,33 @@
+class CfObsBinaryBuilder::Libgdiplus < CfObsBinaryBuilder::BaseDependency
+    def initialize(version)
+      super(
+        "libgdiplus",
+        version,
+        "https://github.com/mono/libgdiplus/archive/#{version}.tar.gz"
```

Comment:
The version is not fetched from a git repo, we can verify a checksum. And depwatcher has support for it afaik.